src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started to work");

```

Comment:
The message 'Excavator started to work' should be more informative and indicate the type of machine. Consider including the class name in the message, for example, 'Excavator is working'.